packages/app/config/babel.prod.js

```diff
@@ -4,33 +4,32 @@ module.exports = {
   // Don't try to find .babelrc because we want to force this configuration.
   babelrc: false,
   presets: [
+    require.resolve('@babel/preset-flow'),
     // Latest stable ECMAScript features
     [
       'env',
       {
         targets: {
           ie: 11,
```

Comment:
I'm thinking that we should replace this target with something sensible for CodeSandbox, like 99%>. I will check the analytics.

Reply:
Great, do that :)